todo-app/src/AppDate.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export class AppDate extends React.Component{
+    render(){
+    let items = this.props.tasks.map((elem, i)=>{
+        return <div>
```

Comment:
Here's the other case of the `key` prop being nested and not on the top-level JSX.